mopidy_local_sqlite/schema.py

```diff
@@ -169,8 +169,7 @@ def load(c):
     user_version = c.execute('PRAGMA user_version').fetchone()[0]
     if not user_version:
         logger.info('Creating SQLite database schema v%s', schema_version)
-        script = os.path.join(sql_dir, 'create-v%s.sql' % schema_version)
-        c.executescript(open(script).read())
+        c.executescript(open(os.path.join(sql_dir, 'schema.sql')).read())
```

Comment:
Not that we'll run on PyPy in the immediate future, but PyPy would issue a warning about a leaking resource here. Alternative version:

``` python
with open(os.path.join(sql_dir, 'schema.sql')) as fh:
    c.executescript(fh.read())
```